### PR TITLE
Refactor display logic & fix browser size issue

### PR DESCRIPTION
**Pull request by zakharvoit**
This PR gets rid of fragments in Chromium implementation, instead we attach ContentViewRenderView directly to our widgets container.

Browser size issue was fixed by updating the ContentViewRenderView's size directly whenever surfaceChanged() is called.

### CODE REVIEW

**Review by duswnchl — APPROVED** [review]
LGTM

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java [hunk 1]
```diff
@@ -4,19 +4,18 @@
 import android.content.Context;
 import android.content.res.Configuration;
 import android.util.Log;
+import android.view.View;
 import android.view.ViewGroup;
 
 import androidx.annotation.NonNull;
 import androidx.fragment.app.FragmentManager;
```
Comment (svillar) [inline app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java:11]: Not needed anymore

**Review by svillar — CHANGES_REQUESTED** [review]
Looking good, I have a couple of comments though

**Review by mshin-wolvic — APPROVED** [review]

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java [hunk 2]
```diff
@@ -146,8 +144,7 @@ public void setExternalVRContext(long externalContext) {
     }
 
     @Override
-    public void setFragmentManager(@NonNull FragmentManager fragmentManager, @NonNull ViewGroup container) {
-        mFragmentManager = fragmentManager;
+    public void setContainerView(@NonNull ViewGroup container) {
```
Comment (svillar) [inline app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java:147]: Nitpicking a bit, but either the attribute name is wrong or the new name is wrong. If it's a container view then the attribute should be `mContainerView`, but if it's a view container then the method should be called `setViewContainer`

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java
Comment (svillar) [inline app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java:1]: As a general comment please fix the typos in the description WRunime -> WRuntime and singature->signature

**Review by svillar — CHANGES_REQUESTED** [review]
Cool, patch is almost ready indeed

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java (hunk 2 above)
Comment (zakharvoit) [reply app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java]: Fixed. It's a container view that is acting as a container for views :) So I went with containerView, IMO it's a more logical name.

File: app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java
Comment (zakharvoit) [reply app/src/common/chromium/com/igalia/wolvic/browser/api/impl/RuntimeImpl.java:1]: Fixed the commit description.

**Review by svillar — APPROVED** [review]
Looks good

**Pull request merged**